src/ResourceManager/Automation/Commands.Automation/Cmdlet/ImportAzureAutomationDscNodeConfiguration.cs

```diff
@@ -31,30 +32,47 @@ public class ImportAzureAutomationDscNodeConfiguration : AzureAutomationBaseCmdl
         private bool overwriteExistingConfiguration;
 
         /// <summary>
+        /// True to create a new build version; false otherwise.
+        /// </summary>
+        private bool _incrementNodeConfigurationBuild; 
+
+        /// <summary>
         /// Gets or sets the source path.
         /// </summary>
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Path to the node configuration .mof to import.")]
+        [Parameter(ParameterSetName = AutomationCmdletParameterSets.ByNodeConfiguration, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Path to the node configuration .mof to import.")]
+        [Parameter(ParameterSetName = AutomationCmdletParameterSets.ByNodeConfigurationBuild, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Path to the node configuration .mof to import.")]
         [ValidateNotNullOrEmpty]
         public string Path { get; set; }
 
         /// <summary>
         /// Gets or sets the configuration name for the node configuration.
         /// </summary>
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of the DSC Configuration to import the Node Configuration under. All Node Configurations in Azure Automation must exist under a Configuration. The name of the Configuration will become the namespace of the imported Node Configuration, in the form of 'ConfigurationName.MofFileName'")]
+        [Parameter(ParameterSetName = AutomationCmdletParameterSets.ByNodeConfiguration, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of the DSC Configuration to import the Node Configuration under. All Node Configurations in Azure Automation must exist under a Configuration. The name of the Configuration will become the namespace of the imported Node Configuration, in the form of 'ConfigurationName.MofFileName'")]
+        [Parameter(ParameterSetName = AutomationCmdletParameterSets.ByNodeConfigurationBuild, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of the DSC Configuration to import the Node Configuration under. All Node Configurations in Azure Automation must exist under a Configuration. The name of the Configuration will become the namespace of the imported Node Configuration, in the form of 'ConfigurationName.MofFileName'")]
         public string ConfigurationName { get; set; }
 
 
         /// <summary>
         /// Gets or sets switch parameter to confirm overwriting of existing configurations.
         /// </summary>
-        [Parameter(Mandatory = false, HelpMessage = "Forces the command to overwrite an existing Node Configuration.")]
+        [Parameter(ParameterSetName = AutomationCmdletParameterSets.ByNodeConfiguration, Mandatory = false, HelpMessage = "Forces the command to overwrite an existing Node Configuration.")]
```

Comment:
@cormacpayne The function of the -Force was to enable overwrite any existing DscNodeConfiguration and the new parameter enabled them to create a new version. Thus -force and -IncrementNodeConfigurationBuild are opposite of each other and could not reside in the same parameter set.